setup

```diff
@@ -1,21 +1,15 @@
-#!/usr/bin/env python3
+#!/usr/bin/bash
```

Comment:
It doesn't seem to work here (macOS).

<details>

```console
$ ./setup
zsh: ./setup: bad interpreter: /usr/bin/bash: no such file or directory
$ which bash
/bin/bash
```

</details>


This leads me to another question: what about Windows users?

Maybe it's time to get rid of `setup` and replace it by a few command line instructions — so we can be more specific in addressing multiple platforms.